base-image/Dockerfile

```diff
@@ -134,4 +134,16 @@ ONBUILD RUN echo "Checking for 'start'..." \
         chmod +x start \
         && cp start /srv/start \
         ; fi
+
+# Copy jupyter_notebook_config.py to /etc/jupyter, which is an action not done
+# with repo2docker.
+ONBUILD USER root
```

Comment:
the first onbuild commands run as root, so if you move this section up in the Dockerfile you don't have to switch back to root for this piece

Reply:
Ah! I put it in the bottom because it could be useful to not break the cache for the other steps, but the cache will always break because of the `COPY .` statement that proceeds all installation steps.

Since I have no clear solution for that, and it would be feature creep to solve it here, I agree and moved it to the root section!